src/screens/WalletConnectApprovalSheet.js

```diff
@@ -355,6 +356,10 @@ export default function WalletConnectApprovalSheet() {
     handled.current = true;
     goBack();
     handleSuccess(true);
+
+    setTimeout(() => {
+      handleReviewPromptAction(ReviewPromptAction.DappConnections);
+    }, 500);
```

Comment:
Will this interrupt wallet connect flow if review prompts and the app needs to direct the user back to the other app?

Reply:
this is a good point, did we confirm this? even if not "interrupt" but is it just funky

Reply:
@jinchung @walmat 
I see once I'm prompted to review, the WC connection modal is disabled, I must interact with the review prompt first in order to connect

![Screenshot 2023-10-20 at 7 44 18 PM](https://github.com/rainbow-me/rainbow/assets/41752922/2cab2ba0-e92f-48de-af90-5451a6a085c2)

Reply:
Yeah was worried about this one. I'm down to remove it from here if that's the play.

Reply:
i'd move it into the walletconnect callback so it's after we call goBack, should also wrap in InteractionManager at that point as well if we do